pkgs/development/libraries/mesa/common.nix

```diff
@@ -1,18 +1,18 @@
-{ lib, fetchurl }:
+{ lib, fetchFromGitLab }:
 # When updating this package, please verify at least these build (assuming x86_64-linux):
 # nix build .#mesa .#pkgsi686Linux.mesa .#pkgsCross.aarch64-multiplatform.mesa .#pkgsMusl.mesa
 # Ideally also verify:
 # nix build .#legacyPackages.x86_64-darwin.mesa .#legacyPackages.aarch64-darwin.mesa
```

Comment:
Should these go in `passthru.tests`?

Reply:
We can't really do that without instantiating a whole bunch of extra nixpkgses.

Reply:
If you took `pkgsi686Linux` etc. as arguments they’d still only be instantiated if you evaluated `passthru.tests`, wouldn’t they?

Reply:
Yeah, ofborg/hydra mostly.

Reply:
We're doing that in intel-media-driver and so far no one complained https://github.com/NixOS/nixpkgs/blob/4dfe798decc582285d0d4e3ec426db524ca1b10f/pkgs/development/libraries/intel-media-driver/default.nix#L59